learning_journal/views/default.py

```diff
@@ -3,18 +3,30 @@
 from datetime import date as Date
 from pyramid.httpexceptions import HTTPFound
 from ..models import Entry
+from learning_journal.security import check_credentials
+from pyramid.security import remember, forget
+from pyramid.security import NO_PERMISSION_REQUIRED
 
 THIS_DIR = os.path.dirname(__file__)
 
 
-@view_config(route_name='home', renderer='templates/index.jinja2')
+@view_config(route_name='home', renderer='templates/index.jinja2', require_csrf=False)
 def home_view(request):
     """Home view handler."""
-    entries = request.dbsession.query(Entry).order_by(-Entry.id).all()
-    entry_list = []
-    for entry in entries:
-        entry_list.append(entry)
-    return {'entries': entry_list}
+    if request.method == 'GET':
+        entries = request.dbsession.query(Entry).order_by(-Entry.id).all()
+        entry_list = []
+        for entry in entries:
+            entry_list.append(entry)
+        return {'entries': entry_list}
+    if request.method == "POST":
```

Comment:
Your home view shouldn't have a check for post method.  Either create a new route/view or use an existing one that does this same function.